_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/pages/_searchapp.scss

```
@@ -0,0 +1,270 @@
+rhdp-search-app {
+
+  .hide {
+    display: none;
+  }
+
+  .show {
+    display: block;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  .loading {
+    background: url("/images/icons/ajax-loader.gif") center 80px no-repeat;
+    min-height: 250px;
+  }
+}
+
+.rhd-c-search-page-header {
+  margin-bottom: 1rem;
+
+  .pf-c-input-group {
+    .pf-c-button.pf-m-danger {
+      &:after {
+        //remove pf after causing blue underline on hover
+        display: none;
+      }
+    }
+  }
+}
+
+rhdp-search-result-count {
+  display: block;
+  margin-bottom: 1em;
+  font-weight: 600;
+  font-size: 1.2em;
+}
+
+rhdp-search-sort-page {
+  display: block;
+  margin-bottom: .5rem;
+  clear: both;
+}
+
+rhdp-search-filters {
+  margin-bottom: 1rem;
+  display: block;
+
+  // Active filter labels (not checkboxes)
+  .activeFilters {
+    rhdp-search-filter-item[active] {
+      display: inline-block;
+
+      .pf-c-label {
+        cursor: pointer;
+        margin: .25rem;
+      }
+    }
+  }
+
+  .title {
+    background-color: var(--pf-global--palette--black-100, #fafafa);
+    color: var(--rhd-theme--color-text, #151515);
+    font-family: RedHatDisplay;
```

```suggestion
  font-family: 'Red Hat Display', sans-serif;
```

`sans-serif` should be added as a fallback font family.